apps/greenhouse/src/components/admin/Layout.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Juno contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useRef } from "react"
+import { useNavigate, useMatches, AnySchema } from "@tanstack/react-router"
+import { AppShell, TopNavigation, TopNavigationItem } from "@cloudoperators/juno-ui-components"
+
+const navigationItems = [
+  {
+    label: "Clusters",
+    value: "/admin/clusters",
+  },
+  {
+    label: "Teams",
+    value: "/admin/teams",
+  },
+  {
+    label: "Plugins",
```

Comment:
To avoid any confusion we might need to renamed to `Plugin Presets` and also `/admin/plugin_presets` since from this entry point we will also display the plugins from the presets...